### ISSUE TITLE
After the SD OTA, %landscape hash doesn't match, Groups is stocked at installing

### ISSUE THREAD

**Issue by pilwex-migsun** [urbit/urbit#5274]
**Describe the bug**
The Software Distribution OTA seems to have been applied, except for the landscape desk. The %landscape hash is not matching (`m1nt8` instead of `e4jfn`). and the Groups tile doesn't work and is stocked at Installing. 

**To reproduce the bug**
Let pilwex-migsun download and apply the OTA.

**System**
 - OS: linux64
 - Binaries version: 1.5
 - `+trouble` output:
```
> +trouble
%bitcoin
  /sys/kelvin:      [%zuse 420]
  base hash:        ~[0v1k.bsql1.30pie.bm8li.tj52j.6l2mo.hcscv.g7ebl.m1h82.quchb.0uoma 0v14.25lr5.4ps7o.2e8qc.j8uo3.7lc4t.a4u4k.k39qk.4e7ff.mlrjc.3eo6o]
  %cz hash:         0v15.309sf.6l1bk.l6bh6.c142e.of3jg.tbna0.qehna.4go0u.ceq25.8kkfo
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          tracking
  source ship:      ~marnus
  source desk:      %bitcoin
  source aeon:      2
  pending updates:  ~
::
%base
  /sys/kelvin:      [%zuse 420]
  base hash:        0v1f.kbord.s5skj.l2210.38out.kgbka.sbkr4.pbuul.vbamp.h57ih.fgbps
  %cz hash:         0v1f.kbord.s5skj.l2210.38out.kgbka.sbkr4.pbuul.vbamp.h57ih.fgbps
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          tracking
  source ship:      ~marnus
  source desk:      %kids
  source aeon:      103
  pending updates:  ~
::
%landscape
  /sys/kelvin:      [%zuse 420]
  base hash:        0v10.406nc.nhrti.jl75k.g1i6j.8s9hr.9qlso.aigie.tre10.m2g48.m1nt8
  %cz hash:         0v10.406nc.nhrti.jl75k.g1i6j.8s9hr.9qlso.aigie.tre10.m2g48.m1nt8
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  [~ ~lander-dister-dozzod-dozzod]
  updates:          tracking
  source ship:      ~marnus
  source desk:      %landscape
  source aeon:      3
  pending updates:  ~
::
%webterm
  /sys/kelvin:      [%zuse 420]
  base hash:        ~[0v1k.bsql1.30pie.bm8li.tj52j.6l2mo.hcscv.g7ebl.m1h82.quchb.0uoma 0v14.25lr5.4ps7o.2e8qc.j8uo3.7lc4t.a4u4k.k39qk.4e7ff.mlrjc.3eo6o]
  %cz hash:         0vj.uts82.9e1g0.avvsq.ih832.l1qi1.0v1uh.o7565.qjiav.ctin2.b2pme
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          tracking
  source ship:      ~marnus
  source desk:      %webterm
  source aeon:      2
  pending updates:  ~
::
%garden
  /sys/kelvin:      [%zuse 420]
  base hash:        ~[0v1k.bsql1.30pie.bm8li.tj52j.6l2mo.hcscv.g7ebl.m1h82.quchb.0uoma 0v14.25lr5.4ps7o.2e8qc.j8uo3.7lc4t.a4u4k.k39qk.4e7ff.mlrjc.3eo6o]
  %cz hash:         0v1e.2h7hs.elq3g.1sdt7.qfga6.ganga.7p95j.aog44.8p5fe.kpr6v.7ai82
  app status:       running
  force on:         ~
  force off:        ~
  publishing ship:  ~
  updates:          tracking
  source ship:      ~marnus
  source desk:      %garden
  source aeon:      2
  pending updates:  ~
::
```

**Additional context**
Restarting the ship didn't help.
Tried `|paues`ing and `|resume`ing %ladnscape at no avail:
```
kiln: %landscape resuming updates
kiln: resetting tracking for %landscape from [~marnus %landscape]
> |resume %landscape
>=
kiln: activated install into %landscape from [~marnus %landscape]
kiln: downloading update for %landscape from [~marnus %landscape]
kiln: finished downloading update for %landscape from [~marnus %landscape]
kiln: remote is identical to %landscape from [~marnus %landscape], skipping
docket: fetching %http glob for %landscape desk
```

**Comment by Fang-** [comment]
> (`m1nt8` instead of `e4jfn`)

`m1nt8` is, as far as I'm aware, latest landscape. Perhaps we need to update the faq? cc @eamsden 

As for the tile being stuck at installing, wonder if that's just a display issue. What happens if you go to `yourship-or-ip.com/apps/landscape/` in the browser directly?

**Comment by pilwex-migsun** [comment]
Nothing, 404

**Comment by Fang-** [comment]
Does the `/apps/grid` page it redirects to load for you?

If not, can you give me the output of `.^(? %gu /=docket=)`? Should be either `%.y` or `%.n`.

If it does load, you could try `|uninstall %landscape` followed by `|install (sein:title our now our) %landscape`. Failing that, I'd like to see the console output you got after running those commands.

**Comment by pilwex-migsun** [comment]
If `/apps/grid` loads? yes it works fine, also BTC and web-terminal.
Uninstalled and the tile was still spinning (cache cleared).
Installed and still the same.
Ran .^ and got %.y
Output of the installation commands:
```
kiln: uninstalling %landscape from [~marnus %landscape]
> |uninstall %landscape
>=
gall: not running %metadata-store yet, got %leave
gall: not running %s3-store yet, got %leave
gall: not running %hark-group-hook yet, got %leave
gall: not running %hark-graph-hook yet, got %leave
gall: not running %invite-store yet, got %leave
gall: not running %group-view yet, got %leave
gall: not running %graph-store yet, got %leave
gall: not running %weather yet, got %leave
gall: not running %dm-hook yet, got %leave
gall: not running %contact-pull-hook yet, got %leave
gall: not running %launch yet, got %leave
gall: not running %contact-store yet, got %leave
gall: not running %contact-store yet, got %watch
gall: not running %contact-pull-hook yet, got %watch
gall: not running %hark-graph-hook yet, got %watch
gall: not running %contact-pull-hook yet, got %watch
gall: not running %contact-store yet, got %watch
gall: not running %hark-graph-hook yet, got %watch
gall: %graph-pull-hook dozing, got %raw-fact
eyre: canceling ~[//http-server/0v2h.tnrhi/4330327650723213045/1]
gall: %graph-pull-hook dozing, got %raw-fact
gall: not running %contact-pull-hook yet, got %watch
gall: not running %contact-store yet, got %watch
gall: not running %hark-graph-hook yet, got %watch


kiln: beginning install into %landscape from [~marnus %landscape]
> |install (sein:title our now our) %landscape
>=
kiln: activated install into %landscape from [~marnus %landscape]
kiln: downloading update for %landscape from [~marnus %landscape]
kiln: finished downloading update for %landscape from [~marnus %landscape]
kiln: remote is identical to %landscape from [~marnus %landscape], skipping
gall: reviving %metadata-store
gall: reviving %contact-store
gall: reviving %chat-store
gall: reviving %graph-store
gall: reviving %group-store
gall: reviving %invite-store
gall: reviving %s3-store
gall: reviving %chat-cli
gall: reviving %chat-hook
gall: reviving %chat-view
gall: reviving %clock
gall: reviving %contact-hook
gall: reviving %contact-pull-hook
gall: reviving %contact-push-hook
gall: reviving %contact-view
gall: reviving %dm-hook
gall: reviving %graph-pull-hook
gall: reviving %graph-push-hook
gall: reviving %group-pull-hook
gall: reviving %group-push-hook
gall: reviving %group-view
gall: reviving %hark-chat-hook
gall: reviving %hark-graph-hook
out=[acked=%.y path=/updates]
subscribe wire not unique
%hark-graph-hook
/graph
[p=~pilwex-migsun q=%graph-store]
error in %hark-graph-hook
%watch-not-unique
! subscribe wire not unique
! %hark-graph-hook
! /graph
! [p=~pilwex-migsun q=%graph-store]
gall: reviving %hark-group-hook
gall: reviving %hark-invite-hook
gall: reviving %invite-hook
gall: reviving %invite-view
gall: reviving %launch
gall: reviving %metadata-hook
gall: reviving %metadata-pull-hook
gall: reviving %metadata-push-hook
gall: reviving %notify
gall: reviving %observe-hook
gall: reviving %sane
gall: reviving %weather
docket: fetching %http glob for %landscape desk
eyre: canceling ~[//http-server/0v2h.tnrhi/16708294728868873868/1]
eyre: canceling ~[//http-server/0v2h.tnrhi/10961054732976631796/1]
```

**Comment by Fang-** [comment]
Oh the `not running yet` printfs are certainly suspicious. But it revived the affected applications afterwards, so I assume you're no longer seeing those particular printfs?

> `docket: fetching %http glob for %landscape desk`

This might fetching take a little bit. Has the groups app started working by now? If not, can you post the output of `:docket +dbug [%state '(~(get by charges) %landscape)']` please? That should tell us if it got stuck fetching the glob, or what else is going on there.

**Comment by pilwex-migsun** [comment]
Still getting `docket: fetching %http glob for %landscape desk` at startup.

Ok, just got a notification from a group so they must be revived. But the tile is still stocked and `apps/landscape/` is 404ed.

```
[~ [docket=[%1 title='Groups' info='A suite of applications to communicate on Urbit' color=0xee.5432 href=[%glob base=%landscape glob-reference=[hash=0v7.597ab.3d6qj.b7a9l.5no45.9dq2g location=[%http url='https://bootstrap.urbit.org/glob-0v7.597ab.3d6qj.b7a9l.5no45.9dq2g.glob']]] image=~ version=[major=1 minor=3 patch=5] website='https://tlon.io' license='MIT'] chad=[%install ~]]]
> :docket +dbug [%state '(~(get by charges) %landscape)']
>=
```

**Comment by pilwex-migsun** [comment]
It's working now. Thank you @Fang- . 

What happened (for reference):
Was still getting 
``
docket: thread cancelled; retrying
%http-request-cancelled
http request was cancelled by the runtime
docket: fetching %http glob for %landscape desk
``
at start up. So, I downloaded the glob and mounted the desk to see if I can upload the glob manually. And then it was resolved some how.

But is there a command to force docket to get the glob over %ames?
